src/main/java/org/gooru/insights/constants/APIConstants.java

```diff
@@ -313,6 +317,10 @@
 	public static final String ALL = "all";
 	
 	public static final String FIELDS_TO_TIME_FORMAT = "totalTimespent";
```

Comment:
@kumaraswamyS , As this fields used in matches for commen functionality we couldn't change the  field name same as value.